src/dotnet/Fable.Tools/State.fs

```diff
@@ -106,20 +129,21 @@ let loadPlugins pluginPaths (loadedPlugins: PluginInfo list) =
             | ex -> failwithf "Cannot load plugin %s: %s" path ex.Message)
     |> Seq.toList
 
-/// Returns an (errors, warnings) tuple
-let parseErrors errors =
-    let parseError (er: FSharpErrorInfo) =
-        let isError, severity =
-            match er.Severity with
-            | FSharpErrorSeverity.Warning -> false, "warning"
-            | FSharpErrorSeverity.Error -> true, "error"
-        isError, sprintf "%s(L%i,%i) : %s FSHARP: %s"
-            er.FileName er.StartLineAlternate er.StartColumn
-            severity er.Message
-    errors
-    |> Array.map parseError
-    |> Array.partition fst
-    |> fun (ers, wns) -> Array.map snd ers, Array.map snd wns
+// --- perhaps not used anymore? ---
+// /// Returns an (errors, warnings) tuple
+// let parseErrors errors =
+//     let parseError (er: FSharpErrorInfo) =
+//         let isError, severity =
+//             match er.Severity with
+//             | FSharpErrorSeverity.Warning -> false, "warning"
+//             | FSharpErrorSeverity.Error -> true, "error"
+//         isError, sprintf "%s(L%i,%i) : %s FSHARP: %s"
+//             er.FileName er.StartLineAlternate er.StartColumn
+//             severity er.Message
+//     errors
+//     |> Array.map parseError
+//     |> Array.partition fst
+//     |> fun (ers, wns) -> Array.map snd ers, Array.map snd wns
```

Comment:
Yes, this is not used at the moment as errors don't throw anymore. This is to keep the compilation going on and being able to offer the full error list at the end. See #806.